### ISSUE TITLE
Failed to load file '/Users/-/Downloads/./Skins/-+Shige+Seoul+(16-9)/count3s.wav'.

### ISSUE THREAD

**Issue by Ooglely** [itdelatrisu/opsu#199]
**Version:** 0.12.1
**Build date:** 2015-10-02 03:05
**OS:** Mac OS X (x86_64)
**JRE:** 1.8.0_101
**Error:** `Failed to load file '/Users/alenavdic/Downloads/./Skins/-+Shige+Seoul+(16-9)/count3s.wav'.`
**Stack trace:**

```
javax.sound.sampled.UnsupportedAudioFileException: could not get audio input stream from input URL
    at javax.sound.sampled.AudioSystem.getAudioInputStream(AudioSystem.java:1155)
    at itdelatrisu.opsu.audio.SoundController.loadClip(SoundController.java:92)
    at itdelatrisu.opsu.audio.SoundController.init(SoundController.java:213)
    at itdelatrisu.opsu.states.Splash$2.run(Splash.java:157)
```

**Comment by iceblade112** [comment]
Is `count3s.wav` perhaps an OGG or MP3 file renamed to .wav?

**Comment by Ooglely** [comment]
I am not sure about that.

**Comment by iceblade112** [comment]
Try to open it with a hex editor, is there text in the header?

**Comment by itdelatrisu** [comment]
Alternatively, upload the file for us to check instead.

**Comment by itdelatrisu** [comment]
Should be resolved in 8655794.
